token/transfer-hook/cli/src/main.rs

```diff
@@ -407,8 +378,9 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 .unwrap()
                 .unwrap();
             let transfer_hook_accounts = arg_matches
-                .get_many::<AccountMeta>("transfer_hook_account")
+                .get_many::<Vec<ExtraAccountMeta>>("transfer_hook_account")
```

Comment:
I don't think this works currently
```suggestion
                .get_many::<Vec<ExtraAccountMeta>>("transfer_hook_accounts")
```